### PR TITLE
Improve air gapped deployment documentation 

### PR DESCRIPTION
**Pull request by rjmateus**
# Description

Improve air gap deployments for Server and proxy.
Correct formating of soma storage procedure, which as rendering `+` signals

# Target branches

Backport targets (edit as needed):

- master
- 4.3

# Links
- This PR tracks issue #<insert spacewalk issue, if any>
- Related development PR #<insert PR link, if any>

### CODE REVIEW

File: modules/installation-and-upgrade/pages/container-deployment/suma/proxy-air-gapped-deployment-suma.adoc [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+. Update the system.
+. Install tools packages and image packages (replace $ARCH$ with the correct architecture)
++
+[source,shell]
+----
+transactional-update pkg install mgradm* mgrctl* suse-manager-5.0-$ARCH$-server-*
+----
++
+. Reboot.
+. Deploy {productname} with mgradm.
```
Comment (admd) [inline modules/installation-and-upgrade/pages/container-deployment/suma/proxy-air-gapped-deployment-suma.adoc:36]:
```suggestion
. Deploy {productname} with mgrpxy.
```

File: modules/installation-and-upgrade/pages/container-deployment/suma/proxy-air-gapped-deployment-suma.adoc [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+== What is air-gapped deployment?
+
+Air-gapped deployment refers to the setup and operation of any networked system that is physically isolated from insecure networks, especially the internet.
+This type of deployment is commonly used in high-security environments such as military installations, financial systems, critical infrastructure, and anywhere sensitive data is handled and must be protected from external threats.
+
+== Deploy with Virtual Machine
+
+The recommended installation method is using the provided {productname} Virtual Machine Image option, since all the needed tools and container images are pre-loaded and will work out of the box.
+
+For more information about installing {productname} Server Virtual Machine, see xref:container-deployment/suma/proxy-deployment-vm-suma.adoc[Deploy Proxy as a Virtual Machine].
```
Comment (admd) [inline modules/installation-and-upgrade/pages/container-deployment/suma/proxy-air-gapped-deployment-suma.adoc:15]:
```suggestion
For more information about installing {productname} Proxy Virtual Machine, see xref:container-deployment/suma/proxy-deployment-vm-suma.adoc[Deploy Proxy as a Virtual Machine].
```

File: modules/installation-and-upgrade/pages/container-deployment/suma/proxy-air-gapped-deployment-suma.adoc [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+.Procedure: Install {productname} on SLE Micro in Air-gapped
+. Install SLE Micro.
+. Bootstrap the Proxy Host OS as a Client on {productname} Server.
+. Update the system.
+. Install tools packages and image packages (replace $ARCH$ with the correct architecture)
++
+[source,shell]
+----
+transactional-update pkg install mgradm* mgrctl* suse-manager-5.0-$ARCH$-server-*
```
Comment (admd) [inline modules/installation-and-upgrade/pages/container-deployment/suma/proxy-air-gapped-deployment-suma.adoc:32]:
```suggestion
transactional-update pkg install mgrpxy* mgrctl* suse-manager-5.0-$ARCH$-proxy-*
```

**Review by jcayouette — APPROVED** [review]
Looks good to me, thank you.

**Pull request merged**